OIDC: Support OAuth org to role mappings (#77683)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

You can disable this default role assignment by setting `role_attribute_strict = true`.
This setting denies user access if no role or an invalid role is returned.

> The `org_roles_attribute_path` configuration option can be used to add roles to other Orgs. See [Org roles mapping examples]({{< relref "#org-roles-mapping-examples" >}})

Reviewer: ```suggestion
You can use the `org_roles_attribute_path` configuration option to add roles to other orgs. For more information, refer to [Org roles mapping examples]({{< relref "#org-roles-mapping-examples" >}}).
```
